src/driver/oracle/OracleDriver.ts

```diff
@@ -342,7 +373,7 @@ export class OracleDriver implements Driver {
      * Escapes a column name.
      */
     escape(columnName: string): string {
-        return `"${columnName}"`;
+        return columnName.toUpperCase();
     }
```

Comment:
Why not just have error messages instead?  This implicitly does something else that might not be expected.

Reply:
Sorry, not sure I follow with error messages instead?

Reply:
If someone passes a column name that's invalid throw an error message instead of implicitly changing the column names to something the user did not intend.

Reply:
Gotcha - that makes sense thanks, by adding the quote marks (pre my change) then we're already changing the column name to something that isn't what the user had initially.

I do have another change as part of this now-closed PR that if a reserved word is used then it will throw a new ReservedWordError.